src/app.js

```diff
@@ -1,7 +1,63 @@
 const express = require('express');
 
+const { Cat } = require('./models');
+
 const app = express();
 
 app.use(express.json());
 
+app.post("/cats", (req, res) => {
+    Cat.create(req.body)
+        .then((cat) => res.status(201).json(cat))
+        .catch((err) => {
+            console.error("error", err);
+            res.status(400).json({ error: "Bad request!" });
+        });
+});
+  
+app.get('/cats', (req, res) => {
+    Cat.findAll({ where: req.query })
+        .then((cat) => res.status(201).json(cat))
```

Comment:
Status 201 is created which is for PATCH/PUT, would be better to use 200 for a GET request.